sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1531,6 +1540,39 @@ public void addContextTag(final @NotNull String contextTag) {
     this.contextTags.add(contextTag);
   }
 
+  /**
+   * Returns whether sending of client reports has been enabled.
+   *
+   * @return true if enabled; false if disabled
+   */
+  public boolean isSendClientReports() {
+    return sendClientReports;
+  }
+
+  /**
+   * Enables / disables sending of client reports.
+   *
+   * @param sendClientReports true enables client reports; false disables them
+   */
+  public void setSendClientReports(boolean sendClientReports) {
+    this.sendClientReports = sendClientReports;
+
+    if (sendClientReports) {
```

Comment:
Do we want to have a check here so we only create a new instance if sending was disabled before?

Reply:
We don't expect users to make many changes to the options. You usually configure them once. I think we don't need to optimize for that.